types/postcss-prefix-selector/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for postcss-prefix-selector 1.14
+// Project: https://github.com/RadValentin/postcss-prefix-selector
+// Definitions by: robertmaier <https://github.com/robertmaier>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/postcss-prefix-selector
+
+interface Options {
+    prefix?: string;
+    exclude?: ReadonlyArray<string | RegExp>;
+    ignoreFiles?: ReadonlyArray<string | RegExp>;
+    includeFiles?: ReadonlyArray<string | RegExp>;
+    transform?: (prefix: Readonly<string>, selector: Readonly<string>, prefixedSelector?: Readonly<string>, file?: Readonly<string>) => string;
```

Comment:
this one seems to be always called with all 4 params:
https://github.com/RadValentin/postcss-prefix-selector/blob/master/index.js#L43-L48
params for callbacks are always optional:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks